### DISCUSSION TITLE
Trakt Authentication Requires a Browser Window Every Time

### DISCUSSION THREAD

**Discussion by teedoff087** [meisnate12/Plex-Meta-Manager#618] [Q&A] [answered]
I'm having trouble with the Trakt authentication in my config.yml. So my authentication goes through just fine: I run PMM, get the code, paste it back into the script and it runs fine. But it's not populating the authorization data in config.yml like it says it's supposed to. 

I'm on macOS Big Sur, so I use launchd to schedule PMM to run twice a day. Ever since I added the Trakt authentication credentials into my config.yml, it no longer updates my collections because it's getting hung up on the Trakt authorization. 

Is there a way to get this to work without requiring a browser window every time?

**Comment by meisnate12** [comment, marked as answer]
There was a trakt bug in 1.14.x have you upgraded to 1.15.0?

**Reply by teedoff087** [reply]
I feel dumb. That fixed it, thank you.
